packages/react-next/src/components/Coachmark/Coachmark.base.tsx

```diff
@@ -34,126 +33,391 @@ export const COACHMARK_ATTRIBUTE_NAME = 'data-coachmarkid';
  * An interface for the cached dimensions of entity inner host.
  */
 export interface IEntityRect {
-  width: number;
-  height: number;
+  width?: number;
```

Comment:
Why are these optional now?

Reply:
I don't recall the precise details. Per the [commit message](https://github.com/MLoughry/office-ui-fabric-react/commit/26478ffa47d9330bf1cd165a3b17b8f3a4e0d2ee), this was to fix a bug (currently still in the OUFR version) where initializing the component with `isCollapsed={false}` would not show the expanded Coachmark.